src/main/java/frc/robot/subsystems/Drivebase.java

```diff
@@ -222,23 +241,10 @@ public void setRamping(double ramping) {
         right.configClosedloopRamp(0);
     }
 
-    public double getRamping() {
```

Comment:
Is this method just unused (and thus unnecessary)?

Reply:
It is unused IIRC and was there for our old MotionMagic code